pkg/run/run.go

```diff
@@ -464,6 +464,8 @@ func (r *RunOptions) PreRunSetup(kclient kubernetes.Interface) error {
 	}
 	log.Infof("Created %s ClusterRoleBinding", pkg.PrivilegedClusterRoleBinding)
 
+	// Create the e2e-dedicated-controller deployment
```

Comment:
did you intend to have something here?

Reply:
maybe it is old code? I cant find

Reply:
iirc I did some cleanup in the last commit